libkodimote/libkodimote.pro

```diff
@@ -3,7 +3,7 @@ contains(QT_VERSION, ^5\\..\\..*) {
     DEFINES += QT5_BUILD
     QT += quick qml
 } else {
-    QT += declarative
+    QT += declarative network
```

Comment:
Good point. Compilation is working by changing `Qt` to `QT` (and fixing the typo in 'concur**r**ent' :)).
Still the change in `imagecache.cpp` is needed, as otherwise `QFutureWatcher` is missing (and the build breaks).